dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -59,6 +61,8 @@
   public static final RemoteRepository CENTRAL =
       new RemoteRepository.Builder("central", "default", "http://repo1.maven.org/maven2/").build();
 
+  private static final List<RemoteRepository> mavenRepositories = Lists.newArrayList(CENTRAL);
```

Comment:
Let's avoid static mutable state if we can. Can the command line parser just create an `ImmutableList` of repositories and put it here, rather than having a mutable one?

Reply:
Updated to
`public static ImmutableList<RemoteRepository> mavenRepositories = ImmutableList.of(CENTRAL);`